src/CSI.IBTA.Administrator/Clients/PlansClient.cs

```diff
@@ -19,20 +19,32 @@ public PlansClient(AuthorizedHttpClient httpClient, ILogger<PlansClient> logger)
             _httpClient.SetBaseAddress("BenefitsServiceApiUrl");
         }
 
-        public async Task<GenericResponse<PlanDto>> GetPlan(int id)
+        public async Task<GenericResponse<List<PlanDto>>> GetPlans(int? userId = null)
         {
-            var response = await _httpClient.GetAsync(string.Format(BenefitsServiceApiEndpoints.Plan, id));
+            var response = await _httpClient.GetAsync(String.Format(BenefitsServiceApiEndpoints.Plan, "?customerId=" + userId));
 
             if (!response.IsSuccessStatusCode)
```

Comment:
As been posted above, could be replaced with response.EnsureSuccessStatusCode(); , will not repeat this message further.